### ISSUE TITLE
Missing in the app

### ISSUE THREAD

**Issue by scharffc** [appdockProject/appDockApp#12]
Videos

Sound

SMS

**Comment by jangerhard** [comment]
@scharffc 

What Twilio number are we using?

**Comment by scharffc** [comment]
@jangerhard 

I sent you out the infos of my account. We should use this. Thanks.

**Comment by jangerhard** [comment]
TL;DR: 
Texting works, but it's not 'safe' according to Twilio. 

To make this slightly easier for ourselves, I've hard-coded the Twilio information into our app, since we are not publishing to the app store. That is that our string.xml contains the SID for the Twilio Account, as well as the secret code. This goes against Twilio's guidelines, as someone could reverse engineer our app (look at github), to find our Twilio information. 
If we wanted to safe guard ourselves, we would have to set up our own server which could communicate with Twilio, and it would require some extra work with little to no payback, since we are only using it for a single purpose. As we are running out of time, I chose this option, but we can change if needed. 

@scharffc

**Comment by scharffc** [comment]
@jangerhard 

I added the twilio account in the file. https://docs.google.com/spreadsheets/d/1c7I_HBXCazxDO0lPRWsWDGeKV9qM_novgjQunKkHHYQ/edit#gid=0
Should we add the key too?

**Comment by jangerhard** [comment]
I think it is fine as is. Unless it needs to be changed. 

Should I also change which Page Facebook is posting to? I need to be admin for that.

**Comment by scharffc** [comment]
Let's leave it for now.
